ictv/renderer/renderer.py

```diff
@@ -101,9 +117,10 @@ def render_template(self, template_name, content, theme_name):
             :param theme_name: The name of the theme.
         """
         rendered_content = self.slide_renderer.__getattr__(template_name)(slide=content)
-        bg = rendered_content.bg if 'bg' in rendered_content else ''
-        themes = Themes.prepare_for_css_inclusion([theme_name])
-        return '<section class="%s" %s>%s</section>' % (' '.join(themes), bg, str(rendered_content))
```

Comment:
Nothing.

With jinja2 this code added a redundant `<section class="" ></section>` which caused the js code in some plugins to fail.

The section became redundant since the template heritage had to be done in a different way with jinja2 as the webpy `base` argument has no equivalent in jinja2. 
(https://webpy.readthedocs.io/en/latest/api.html#web.template.Render)

The `section` tags are now already included in `rendered_content`.

To be fair, I struggled with this one. Some further bugs might appear and some plugins might need minor javascript revisions.